OHM-Trade-Agent-v1/deploy/remote/configure-opip-learning-reader.sh

```
@@ -11,7 +11,224 @@ EXPORT_ROOT="/var/lib/opip-learning-export"
 READER_STATE_ROOT="/var/lib/opip-learning-reader"
 FORCED_COMMAND="/usr/local/sbin/opip-learning-read-export"
 
-if [[ -z "$PUBLIC_KEY" || "$PUBLIC_KEY" != ssh-ed25519\ * || -z "$SOURCE_CIDR" ]]; then
+contains_control_character() {
+  local value="${1:-}"
+  [[ "$value" == *
```

Shellcheck (0.11.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)